src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,19 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        int result = 0;
+        if (a) {
```

Comment:
read [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html) attentively
- Use only boolean operators without if-else or ternary operator.

Reply:
read smth about XOR operation and `||` and `&&`

Reply:
Did it! Happy because it wasn't easy for me :(